x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/apm_jsbase/ml/abnormal_span_durations_jsbase.json

```diff
@@ -0,0 +1,41 @@
+{
+  "job_type": "anomaly_detector",
+  "groups": [
+    "apm"
+  ],
+  "description": "APM JSBase: Looks for spans that are taking longer than usual to process.",
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "high_mean(\"span.duration.us\") partitionfield=\"span.type\"",
```

Comment:
@blaklaybul could you help us come up with some more user friendly descriptions for the detectors in these two modules? They all seem to just use the default strings created by the UI.

Reply:
will do.